scalableminds/webknossos
# docker-compose: rm mongo residue, add persistent pg-dev service

### Steps to test:
- run `USER_UID=$(id -u) USER_GID=$(id -g) docker-compose up dev`, verify that the database is persisted

related to #2447 

------
